lib/ansible/modules/cloud/ovirt/ovirt_network.py

```diff
@@ -213,18 +214,23 @@ def __init__(self, network_id, cluster_network, *args, **kwargs):
         super(ClusterNetworksModule, self).__init__(*args, **kwargs)
         self._network_id = network_id
         self._cluster_network = cluster_network
+        self._old_usages = []
+        self._cluster_network_service = get_entity(self._service.network_service(network_id))
```

Comment:
This is not service so please rename the variable more reasonable. It can be confusing.